app/src/main/java/com/lionheart/presentation/search/SearchFragment.kt

```diff
@@ -16,15 +17,26 @@ class SearchFragment : BindingFragment<FragmentSearchBinding>(R.layout.fragment_
     }
 
     private fun initAdapter() {
-        searchCategoryAdapter = SearchCategoryAdapter()
+        searchCategoryAdapter =
+            SearchCategoryAdapter { searchCategory -> intentToSearchDetail(searchCategory) }
 
         searchCategoryAdapter?.submitList(viewModel.categoryList)
 
         binding.rvSearchCategory.adapter = searchCategoryAdapter
     }
 
+    private fun intentToSearchDetail(searchCategory: SearchCategory) {
+        val intent = Intent(activity, SearchDetailActivity::class.java)
+        intent.putExtra(SEARCH_CATEGORY, searchCategory)
```

Comment:
```suggestion
        val intent = Intent(activity, SearchDetailActivity::class.java).apply {
            putExtra(SEARCH_CATEGORY, searchCategory)
        }
```
scope function을 적극 활용해봅시당!